contrib/sip_proxy/filters/network/source/router/router_impl.h

```diff
@@ -241,7 +241,7 @@ class TransactionInfo : public std::enable_shared_from_this<TransactionInfo>,
   std::shared_ptr<UpstreamRequest> getUpstreamRequest(const std::string& host) {
     try {
       return tls_->getTyped<ThreadLocalTransactionInfo>().upstream_request_map_.at(host);
-    } catch (std::out_of_range) {
+    } catch (std::out_of_range const&) {
```

Comment:
This would be a great candidate for changing upstream too?

Reply:
This *is* from upstream, it's a cherry-pick. It is a separate commit in this PR.

Reply:
ah I see, sorry, missed that.